docs/visual-testing/integrations/webdriverio.md

```diff
@@ -177,12 +177,12 @@ Those ignored components are specified when requesting a new snapshot.
 Example:
 
 ```ts
-    await browser.sauceVisualCheck('Inventory Page', {
-        ignore: [
-            // addBackPackToCartButton will be ignored
-            InventoryPage.addBackPackToCartButton,
-        ],
-    });
+await browser.sauceVisualCheck('Inventory Page', {
```

Comment:
Based on other examples you modified in this pull request I see all example codes start with an additional tab at the beginning. Any reason why you removed it from here?

Reply:
across our docs there is two approaches, one with a tab and one without. IMO we should stick to the second one in all places